crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -643,6 +644,32 @@ fn handle_trailing_end_of_line_condition_comment<'a>(
     CommentPlacement::Default(comment)
 }
 
+/// Handles end of line comments after the `:` of an except clause
+///
+/// ```python
+/// try:
+///    ...
+/// except: # comment
+///     pass
+/// ```
+///
+/// It attaches the comment as dangling comment to the enclosing except handler.
+fn handle_trailing_end_of_line_except_comment<'a>(
+    comment: DecoratedComment<'a>,
+    _locator: &Locator,
+) -> CommentPlacement<'a> {
+    let AnyNodeRef::ExceptHandlerExceptHandler(_) = comment.enclosing_node() else {
+        return CommentPlacement::Default(comment);
+    };
+
+    // Must be an end of line comment
+    if comment.line_position().is_own_line() {
+        return CommentPlacement::Default(comment);
+    }
+
+    return CommentPlacement::dangling(comment.enclosing_node(), comment);
```

Comment:
Can you add a test that has end of line comments that should belong to a statement in the except handler

```python
try:
	...
except ex: 
	a = 10 # trailing comment
```

I believe this implementation will incorrectly attach all comments to the except handler

Reply:
ah yes, of course. fixed